modules/statistics/php/widgets.class.inc

```diff
@@ -236,17 +229,16 @@ class Widgets extends \NDB_Page implements ETagCalculator
         $rv['male_percent']   = $recruitmentTarget ?
             round($totalMales / $recruitmentTarget * 100) :
             null;
-        if ($totalRecruitment > $recruitmentTarget) {
+        if ($totalRecruitment >= $recruitmentTarget) {
             $rv['surpassed_recruitment'] = "true";
-
-            $rv['female_full_percent'] = $totalRecruitment ?
-                round($totalFemales / $totalRecruitment * 100) :
-                null;
-
-            $rv['male_full_percent'] = $totalRecruitment ?
-                round($totalMales / $totalRecruitment * 100) :
-                null;
         }
+        $rv['female_full_percent'] = $totalRecruitment ?
```

Comment:
Not blocking because its the way it's currently done and this is just optimizing but this is going to have to change to not hardcode a binary sex assumption.